app/models/course/discussion/topic.rb

```diff
@@ -7,6 +7,10 @@ class Course::Discussion::Topic < ActiveRecord::Base
 
   accepts_nested_attributes_for :posts
 
+  def to_partial_path
+    'course/discussion/topic'.freeze
```

Comment:
no, because we still run on 2.2 on staging and production. it's harmless anyway.